bidding_auction_services_onboarding_self_serve_guide.md

```diff
@@ -46,12 +46,24 @@ for [enrolling with Coordinators][14].
 
 ### Buyer
 
-  * Refer to [spec for DSP][28].
-  * Develop [generateBid][29]() for bidding.
-  * Develop [reportWin][30]() for event level reporting.
-  * Setup [buyer's Key/Value service][31].
-  * If your Key/Value server supports filtering of interest groups, refer to this
-    section and [metadata forwarding][25].
+Buyers could configure B&A to support PA (both for Chrome and Android traffic) and/or [PAS][120]
```

Comment:
Would it be possible to expand Protected Audience (PA) and continue using PA thereafter? PA acronym is not expanded before this point.

(Background / Context: When the self-serve explainer was published PAS was not public / explainer was not published. Hence we couldn't fork PAS and PA. I do plan to do an small update in this explainer later explaining Protected Auctions.)

Reply:
Done, I expanded PAS too. Thanks!